specification/attributes/string_handling.md

```diff
@@ -0,0 +1,31 @@
+# String Handling
+
+Columns that capture String values conforming to specified requirements foster data integrity, interoperability, and consistency, improve data analysis and reporting, and support more reliable data-driven decision-making.
+
+All columns capturing a String value, defined in the [FOCUS](#glossary:finops-cost-and-usage-specification) specification, MUST follow the requirements listed below. Custom String value capturing columns SHOULD adopt the same requirements over time.
+
+## Attribute ID
+
+StringHandling
+
+## Attribute Name
+
+String Handling
+
+## Description
+
+Requirements for String-capturing columns appearing in billing data.
+
+## Requirements
+
+* String values MUST maintain the original casing, spacing, and other relevant consistency factors as specified by providers and end-users.
+* Changes to mutable string values (e.g., resource names) MUST be accurately reflected in charges incurred after the string value change and MUST NOT alter records incurred before the change, preserving data integrity and auditability for past billing periods.
+* Immutable string values that refer to the same entity (e.g., resource identifiers, region identifiers, etc.) MUST remain consistent and unchanged across all billing periods.
+
+## Exceptions
+
+None
```

Comment:
In response to suggestion https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/396/files#r1567531933 by @cnharris10 

```suggestion
* When a record is provided after a change to a mutable string value and the [ChargeClass](#chargeclass) is "Correction", the record MAY contain the altered value.
```